charts/linstor-cluster/values.yaml

```diff
@@ -0,0 +1,68 @@
+# Definition of LinstorCluster
+# Ref https://github.com/piraeusdatastore/piraeus-operator/blob/v2/docs/reference/linstorcluster.md
+linstorCluster:
+  # Linstor cluster name
+  name: linstorcluster
+#  apiTLS:
+#    certManager:
+#      name: linstor-api-ca
+#      kind: Issuer
+#  internalTLS:
+#    certManager:
+#      name: linstor-internal-ca
+#      kind: Issuer
```

Comment:
> Just wanted to get your opinion on generating certs directly with helm. If its not worth it in your opinion, I will just merge it as-is.

In my opinion, generating certificates in Helm is a rather ugly solution, and I would not recommend anyone to use it.
But I like idea that chart can be expanded later. When someone would need different method, we can simple introduce a new option:
```
method: <cert-manager|helm|openshift|etc>
```